components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -338,56 +255,10 @@ export default function () {
-                                    <img
-                                        alt=""
-                                        className="w-14"
-                                        role="presentation"
-                                        src={isDark ? PrebuildLogsEmptyDark : PrebuildLogsEmpty}
-                                    />
-                                    <h3 className="text-center text-lg text-gray-500 dark:text-gray-50 mt-4">
-                                        No Recent Prebuild
-                                    </h3>
-                                    <p className="text-center text-base text-gray-500 dark:text-gray-400 mt-2 w-64">
-                                        Edit the project configuration on the left to get started.{" "}
-                                        <a className="gp-link" href="https://www.gitpod.io/docs/config-gitpod-file/">
-                                            Learn more
-                                        </a>
-                                    </p>
-                                </div>
-                            )
-                        )}
-                    </div>
-                    <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-600 flex space-x-2">
-                        {prebuildWasTriggered && <PrebuildInstanceStatus prebuildInstance={prebuildInstance} />}
-                        <div className="flex-grow" />
-                        {prebuildWasTriggered && prebuildInstance?.status.phase !== "stopped" ? (
-                            <button
-                                className="danger flex items-center space-x-2"
-                                disabled={
-                                    prebuildWasCancelled ||
-                                    (prebuildInstance?.status.phase !== "initializing" &&
-                                        prebuildInstance?.status.phase !== "running")
-                                }
-                                onClick={cancelPrebuild}
-                            >
-                                <span>Cancel Prebuild</span>
-                            </button>
-                        ) : (
-                            <button disabled={isDetecting} className="secondary" onClick={buildProject}>
-                                Run Prebuild
-                            </button>
-                        )}
-                        <button disabled={isDetecting && buttonNewWorkspaceEnabled} onClick={onNewWorkspace}>
-                            New Workspace
-                        </button>
-                    </div>
+                <div className="h-20 px-6 bg-gray-50 dark:bg-gray-800 border-gray-200 dark:border-gray-600 flex space-x-2">
+                    <button disabled={isDetecting && buttonNewWorkspaceEnabled} onClick={onNewWorkspace}>
+                        New Workspace
```

Comment:
**suggestion**: What do you think of moving the action button to the right following the pattern we already use across the product for having the primary action on the outer right side of things?

| BEFORE | AFTER |
|-|-|
| <img width="801" alt="config-before" src="https://user-images.githubusercontent.com/120486/160588401-9d951433-da3f-47e8-a184-3cceef54c2af.png"> | <img width="801" alt="config-after" src="https://user-images.githubusercontent.com/120486/160588406-7f098695-009f-4ae2-95b9-8fc56a842e31.png"> |